src/features/Calendar/Calendar.scss

```diff
@@ -0,0 +1,79 @@
+.calendar {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+  grid-template-rows: 1.6rem 9rem repeat(4, 9rem);
+  gap: 0.3rem;
+  width: 100%;
+
+  &__week-days {
+    display: flex;
+    height: min-content;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 1rem;
+  }
+
+  &__item {
+    display: flex;
+    flex-direction: column;
+    min-width: 10.5rem;
+    padding: 0 0.3rem;
+    background-color: #F9FBFF;
```

Comment:
kolor tez powinein być zaciągany ze zmiennej  poniżej to samo

Reply:
prawda prawda